File: backend/src/main/java/com/givers/repository/entity/Comment.java
package com.givers.repository.entity;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Document 
@Data 
@AllArgsConstructor
@NoArgsConstructor
public class Comment {
	@Id
	private String id;
	private String content;
	private String owner;
	private String causeId;
}

File: frontend/src/app/comment.ts
import { ThrowStmt } from '@angular/compiler';
import { Cause } from './cause';

export class Comment {
    id: string;
    content: string;
    owner: string;
    causeId: string;

    constructor(
        id: string,
        owner: string,
        cause: string,
        content: string
    ) {
        this.id = id;
        this.owner = owner;
        this.causeId = cause;
        this.content = content;
    }
}
